core/trino-spi/pom.xml

```diff
@@ -205,6 +205,13 @@
                                     <new>method void io.trino.spi.connector.ConnectorViewDefinition.ViewColumn::&lt;init&gt;(java.lang.String, io.trino.spi.type.TypeId, java.util.Optional&lt;java.lang.String&gt;)</new>
                                     <justification>Add support for view column comments</justification>
                                 </item>
+                                <item>
+                                    <ignore>true</ignore>
+                                    <code>java.method.numberOfParametersChanged</code>
+                                    <old>method void io.trino.spi.connector.ConnectorTableLayout::&lt;init&gt;(io.trino.spi.connector.ConnectorPartitioningHandle, java.util.List&lt;java.lang.String&gt;)</old>
+                                    <new>method void io.trino.spi.connector.ConnectorTableLayout::&lt;init&gt;(io.trino.spi.connector.ConnectorPartitioningHandle, java.util.List&lt;java.lang.String&gt;, boolean)</new>
+                                    <justification>Add multipleWritersPerPartition flag in ConnectorTableLayout</justification>
```

Comment:
Remove. Make the change backwards compatible.
This will also help the reader understand how the new flag defaulting happens (per my comment in the other PR)